lib/src/model/typedef.dart

```diff
@@ -38,6 +38,8 @@ class Typedef extends ModelElement
   @override
   String get filePath => '${library.dirName}/$fileName';
 
+  FunctionTypedef get asCallable => this as FunctionTypedef;
```

Comment:
ah, this is for the cast, correct?  I actually had a brief moment where I wondered whether this would be needed during the implementation.

Reply:
Correct. I'd love to consider an extension to our Mustache syntax to allow for casts... there might not be anything better than this silly property, but I think there may be a simple syntax which may also allow us to reduce the number of getters-for-the-sake-of-mustache.